explainaboard/tests/test_machine_translation.py

```diff
@@ -0,0 +1,33 @@
+import pathlib
+import os
+import unittest
+from explainaboard.tasks import TaskType
+from explainaboard import FileType, Source, TaskType, get_loader, get_processor
+
+artifacts_path = os.path.dirname(pathlib.Path(__file__)) + "/artifacts/"
+class TestMachineTranslation(unittest.TestCase):
+
+    def test_generate_system_analysis(self):
+        """TODO: should add harder tests"""
+
+        path_data = artifacts_path+ "test-mt.tsv"
+        loader = get_loader(TaskType.machine_translation, Source.local_filesystem, FileType.tsv, path_data)
+        data = loader.load()
+
+        metadata = {"task_name": TaskType.machine_translation.value,
+                    "dataset_name":"ted_multi",
```

Comment:
So far, we haven't token the languages into account, while when introducing the mt task, we probably need to consider it. (it seems that some evaluation metrics require the information of languages)